manticore/utils/emulate.py

```diff
@@ -248,7 +248,10 @@ def _step(self, instruction):
         saved_PC = self._cpu.PC
 
         try:
-            self._emu.emu_start(self._cpu.PC, self._cpu.PC + instruction.size, count=1)
+            pc = self._cpu.PC
+            if self._uc_mode == UC_MODE_THUMB:
```

Comment:
Can you show a minimal example of this issue?

Also, is there any place we could move it so the general code isn't mixed with cpu-specific code? ;/